packages/phoenix-evals/pyproject.toml

```diff
@@ -22,10 +22,38 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
+  "openinference-semantic-conventions",
```

Comment:
this feels odd - can we leave these functions just in the main phoenix?

Reply:
I agree this feels odd, we can leave this in the main Phoenix module but then we'd have a hard dependency on the `evals` module inside of Phoenix, which we currently don't have. I think I'm going to punt on this for now

Reply:
Okay, let's file a ticket for this. This is super strange and we also probably want the dependency to be harder

Reply:
Tracked by #2356